src/main/webapp/script.js

```diff
@@ -90,53 +113,61 @@ async function displayOverallResults(url) {
  * @param {Array<sentimentBucket>} sentimentBucketList
  */
 function displaySentimentBucketChart(sentimentBucketList) {
-  const CommentSentimentTable = new google.visualization.DataTable();
-  CommentSentimentTable.addColumn('string', 'Sentiment Range');
-  CommentSentimentTable.addColumn('number', 'Comment Count');
-  CommentSentimentTable.addColumn(
-      {'type': 'string', 'role': 'tooltip', 'p': {'html': true}});
-
-  for (i = 0; i < sentimentBucketList.length; i++) {
-    currentSentimentBucket = sentimentBucketList[i];
-    rangeAsString = convertRangeToString(
-        currentSentimentBucket.intervalRange);
-    highestMagnitudeComments = currentSentimentBucket.topNComments;
-
-    CommentSentimentTable.addRow([rangeAsString,
-      currentSentimentBucket.frequency,
-      toTooltipString(highestMagnitudeComments)]);
+  try {
+    const CommentSentimentTable = new google.visualization.DataTable();
+    CommentSentimentTable.addColumn('string', 'Sentiment Range');
+    CommentSentimentTable.addColumn('number', 'Comment Count');
+    CommentSentimentTable.addColumn(
+        {'type': 'string', 'role': 'tooltip', 'p': {'html': true}});
+
+    for (i = 0; i < sentimentBucketList.length; i++) {
+      currentSentimentBucket = sentimentBucketList[i];
+      rangeAsString = convertRangeToString(
+          currentSentimentBucket.intervalRange);
+      highestMagnitudeComments = currentSentimentBucket.topNComments;
+
+      CommentSentimentTable.addRow([rangeAsString,
+        currentSentimentBucket.frequency,
+        toTooltipString(highestMagnitudeComments)]);
+    }
+
+    const options = {
+      'title': 'Comment Sentiment Range',
+      'width': CHART_WIDTH,
+      'height': CHART_HEIGHT,
+      'bar': {groupWidth: '100'},
+      'tooltip': {isHtml: true},
+    };
+
+    const view = new google.visualization.DataView(CommentSentimentTable);
+    const chart = new google.visualization.ColumnChart(
+        document.getElementById('chart-container'));
+    chart.draw(view, options);
+  } catch (err) {
```

Comment:
I think we should remove the try-catch blocks in `displaySentimentBucketChart` and `displayWordCountChart`.

All that is added by having them is that we can signify to the user whether the user they're encountering is caused by the display of the word cloud or caused by the display of a "chart". I'd argue that the UI doesn't need to be concerned with informing the user specifically what went wrong, and instead just needs to let them know that their request wasn't successful.

In addition, this approach has the downside of swallowing the original error (assuming that it happened within either of the two methods).